### PR TITLE
[DOCFIX] Update `Metrics-System.md`

### PR DESCRIPTION
**Pull request by jenoudet**
see title

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -23,87 +23,59 @@ The following instances are currently supported:
...
 * `MetricsServlet`: Adds a servlet in Web UI to serve metrics data as JSON data.
-* `PrometheusMetricsServlet`: Adds a servlet in Web UI to serve metrics data in Prometheus format
 
-The metrics system is configured via a configuration file that Alluxio expects to be present at `$ALLUXIO_HOME/conf/metrics.properties`. 
-A custom file location can be specified via the `alluxio.metrics.conf.file` configuration property. 
-Alluxio provides a `metrics.properties.template` under the `conf` directory which includes all configurable properties
-and guidance on how to specify each property. 
+The metrics system is configured via a configuration file that Alluxio expects to be present at `$ALLUXIO_HOME/conf/metrics.properties`.
+A custom file location can be specified via the `alluxio.metrics.conf.file` configuration property.
+Refer to `$ALLUXIO_HOME/conf/metrics.properties.template` for all possible sink specific configurations.
```
Comment (LuQQiu) [inline docs/en/operation/Metrics-System.md:51]: add parenthesis around ${ALLUXIO_HOME}/conf?

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for improving the doc and adding the Datadog Web UI with Prometheus

File: docs/en/operation/Metrics-System.md (hunk 1 above)
Comment (jenoudet) [reply docs/en/operation/Metrics-System.md]: fixed

**Comment by LuQQiu** [comment]
lol why even checkstyle is queued....

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
@Xenorith cherry-pick

**Pull request merged**